### ISSUE TITLE
Validate 9-digit Martinique numbers starting with 69652 (MQ, +596)

### ISSUE THREAD

**Issue by junpanpan** [googlei18n/libphonenumber#1479]
Please read the "guidelines for contributing" (linked above) and fill in the template below.

Country/region affected (e.g., "US"): SE

Example number(s) affected ("+1 555 555-1234"): 00596696526470

The phone number range(s) to which the issue applies ("+1 555 555-XXXX"): 

The type of the number(s) ("fixed-line", "mobile", "short code", etc.): 

The cost, if applicable ("toll-free", "premium rate", "shared cost"): 

Supporting evidence (for example, national numbering plan, announcement from mobile carrier, news article): **IMPORTANT - anything posted here is made public. Read the guidelines first!** 

[link to demo](http://libphonenumber.appspot.com/phonenumberparser?number=00596696526470&country=SE)

**Comment by kkeshava** [comment]
Hi,

Thank you for taking the time to improve libphonenumber!

We were able to connect to 00596696526470, but country code +596 belongs to Martinique (MQ) and not Sweden (SE). Did you have a typo? We already support +596 696 5[013] as mobile numbers in MQ.

We can't find official evidence for the actual ranges that are valid. It seems like mobile number, could you please confirm and also share carrier details for this range?

Please share any official or supporting evidence (for example, a national numbering plan, announcements from mobile carriers, news articles) you may have so that we can update the range and the carrier details as well. See our [guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md#provide-supporting-evidence).

**Comment by junpanpan** [comment]
Hi, 
I even tried with CC is MQ
Phone Number entered: +00596696526470
defaultCountry entered: MQ

You app still complain it's invalid.  But this is a valid MSISDN.

**Comment by junpanpan** [comment]
What you mean : We already support +596 696 5[013] as mobile numbers in MQ.
?

**Comment by kkeshava** [comment]
@junpanpan ,
In the demo 'Default Country' means the country from which the number is dialed. So if you enter MQ as default country no need not to mention the international prefix (00), as the source and the destination countries are same.

We do not support MQ number +596 696 526 470 currently. Could you please confirm if +596 696 526 470 is mobile / land line? If it is mobile then would you be able to provide the carrier details? 
We need valid evidence to support this number. Please share any official/supporting evidence (for example, a national numbering plan, announcements from mobile carriers, news articles) that you may have so that we can update the range and the carrier details as well. See our guidelines.

Reply to your second query - Currently we are supporting few mobile numbers for MQ which starts with  +596 696 50, +596 696 51 and +596 696 53. As the number provided by you starts with +596 696 52 it might be possible even this a mobile number (as the ranges are close). However we need valid evidence to add the number as mobile.

Once the issue is fixed then the number +596 696 526 470 will become valid.

**Comment by kkeshava** [comment]
Hi,
This issue is blocked on your or other GitHub contributors' input. Please help us find evidence that +596 696 526 470 is valid, or provide an example number from a business website or similar. Otherwise we don't have enough information to fix the issue and may have to close it as infeasible.

**Comment by andyst** [comment]
@junpanpan 
- As mentioned, please specify whether this is a mobile, land-line or some other kind of phone number. Please include evidence for this classification.

- I'm curious, which application or evidence indicated this phone number is associated with Sweden (SE)?

**Comment by keghani** [comment]
In the [ITU communication of 21.V.2005](https://www.itu.int/dms_pub/it-t/oth/02/02/T02020000860001MSWE.doc), we find:

```nomarkdown
International dialling format for the fixed service:  	+596 596 XX XX XX
International dialling format for the mobile service:  	+596 696 XX XX XX
```

This is also the distinction we make in the current metadata:

```nomarkdown
      <fixedLine>
        <nationalNumberPattern>
          596(?:
            0[2-5]|
            [12]0|
            3[05-9]|
            4[024-8]|
            [5-7]\d|
            89|
            9[4-8]
          )\d{4}
        </nationalNumberPattern>
        <possibleLengths national="9"/>
        <exampleNumber>596301234</exampleNumber>
      </fixedLine>
      <mobile>
        <!-- The prefix 69653 is a valid number which was found online in few websites. -->
        <nationalNumberPattern>
          696(?:
            [0-479]\d|
            5[013]|
            8[0-689]
          )\d{4}
        </nationalNumberPattern>
        <possibleLengths national="9"/>
        <exampleNumber>696201234</exampleNumber>
      </mobile>
```

Given that, and that @kkeshava dialled the provided number, we can assume that this issue is about a mobile range.

We've accepted this issue and will fix this in an upcoming release.

**Comment by junpanpan** [comment]
thanks, which release will fix this ?

**Comment by keghani** [comment]
An upcoming one :) We have a large backlog and treat validation issues with highest priority, but it may not make the cut. We'll announce on this thread when it's fixed, thanks for understanding!

**Comment by penmetsaa** [comment]
This is fixed in release 8.1.0
https://libphonenumber.appspot.com/phonenumberparser?number=00596696526470&country=MQ
